test/parallel/test-typescript-eval.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
Can all the new tests please go under `test/es-module`? It’s very convenient for those of us working on modules stuff to be able to run `tools/test.py --mode=release test/es-module` to run all the modules tests without needing to run the full set of parallel tests.